.github/workflows/pr-labeler.yml

```
@@ -0,0 +1,20 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target
+
+jobs:
+  labeler:
+    permissions:
+      contents: read
+      pull-requests: write
```

nit: I prefer we set stricter permissions on top level

```suggestion
permissions:
  contents: read

jobs:
  labeler:
    permissions:
      pull-requests: write
```